libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -1353,6 +1354,10 @@ void AP_Logger::io_thread(void)
         last_run_us = AP_HAL::micros();
 
         FOR_EACH_BACKEND(io_timer());
+
+        if (++counter % 4 == 0) {
```

Comment:
Why isn't this time based?

Reply:
the timing is not critical, and the counter is a bit simpler